webpack.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  entry: ['./src/index.js'],
+  output: {
+    path: __dirname,
+    publicPath: '/',
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [{
+      exclude: /node_modules/,
+      loader: 'babel-loader',
+      query: {
+        presets: ['@babel/react', '@babel/env']
```

Comment:
Hmm wydaje mi się, że to powinno wyglądać tak jak css czyli
```
{
      test: /\.{js,jsx}$/,
      exclude: /node_modules/,
      use: {
        loader: 'babel-loader'
      }
}
```
Wydaje mi się też, że nie potrzebujesz `query` bo to masz sprecyzowane w pliku `.babelrc`

Reply:
zmienilem, w sumie racja z tym query, ale test musialem dac `test: /\.(js|jsx)$/` bo ten co podales nie dzialal ;p

Reply:
Yup źle napisałem :P